packages/js/src/Modules/Verto/index.ts

```diff
@@ -20,7 +20,69 @@ export default class Verto extends BrowserSession {
     return isValidOptions(this.options);
   }
 
+  /**
+   * Makes a new outbound call.
+   *
+   * This method receives an object `options` with the following properties:
+   *
+   * @param options Options object for a new call.
+   * @param options.destinationNumber Extension to dial.
+   * @param options.callerNumber Number to use as the caller ID when dialling out to a phone number.
+   * @param options.callerName Name to use as the caller ID name when dialling out to a phone number.
```

Comment:
Looking at Telnxy support guides and articles, it looks like we use the US English version of `dialing`. Also, since you can dial to a SIP URI, I would use "destination" or just omit "phone number":
```suggestion
   * @param options.callerNumber Number to use as the caller ID when dialing out to a destination.
   * @param options.callerName Name to use as the caller ID name when dialing out to a destination.
```